cmd/clusterctl/cmd/root.go

```diff
@@ -45,7 +50,13 @@ var RootCmd = &cobra.Command{
 
 func Execute() {
 	if err := RootCmd.Execute(); err != nil {
-		// TODO: print error stack if log v>0
+		if *verbosity >= 5 {
```

Comment:
Should we also check that verbosity is not nil?

Reply:
+1 Guess it doesn't hurt to do a nil check even though the default is set to 0.

Reply:
done

Reply:
PTAL when having some time :)